Tests/APITesters/SwiftAPITester/SwiftAPITester/OfferingAPI.swift

```diff
@@ -29,6 +29,10 @@ func checkOfferingAPI() {
     var pPack: Package? = off.package(identifier: "")
     pPack = off.package(identifier: nil)
     let package: Package? = off[""]
+    let metadata: [String: Any] = off.metadata
+    let metadataString: String = off.getMetadataValue(for: "", defaultValue: "")
+    let metadataInt: Int = off.getMetadataValue(for: "", defaultValue: 0)
```

Comment:
I know you were following the pattern here, but it's leftover from the first implementation which was kind of messy.
We should get rid of all these `print` calls.

To avoid increasing the noise, I'd change these to:
```suggestion
    let _: String = off.getMetadataValue(for: "", defaultValue: "")
    let _: Int = off.getMetadataValue(for: "", defaultValue: 0)
```